CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Docs
 
+- Fix example code in `Get started with Druid` chapter of book ([#1698] by [@ccqpein])
```

Comment:
You have to add to the list of links at the bottom of the file for the `[#1698]` and `[@ccqpein]` shortcuts to work

Reply:
Thanks for notifying me, let me change it now.

Reply:
Just done. Please check it out.